tests/test_webtests.py

```diff
@@ -127,15 +120,6 @@ def test_sees_projects_in_her_dashboard(self):
         res = self.app.get('/myprojects/', auth=self.user.auth)
         assert_in('Projects', res)  # Projects heading
 
-    def test_logged_in_index_route_renders_home_template(self):
```

Comment:
These tests were failing on travis with a ` AppError: Bad response: 404 NOT FOUND (not 200 OK or 3xx redirect for http://localhost/)`  https://travis-ci.org/pattisdr/osf.io/jobs/407731240

I assumed this was because this route is now loading in Ember and better tested there.